### ISSUE TITLE
Update HTML components to MUI: ./src/pages/Home.js

### ISSUE THREAD

**Issue by JackHaeg** [hackforla/VRMS#1717]
### Overview
Review all components in`./src/pages/Home.js` and replace all standard HTML components with applicable MUI components.

### Action Items
- [x] Review components in:`./src/pages/Home.js` and search for any standard HTML components.
  - _IF standard HTML components are found in the file:_ 
    - [x] Replace these standard HTML components with applicable MUI components. 
      - [x] Ensure select field integrates changes from the `Selection Field` details below
    - [x] Make a PR for this file

### Selection Field

This component contains a selection field with selection options. Please reveiw MUI's [Selection Component](https://mui.com/material-ui/react-select/) to ensure that "Select One" is displayed by default on page load.

As it currently stands, the old select style is adding a "-- Select One --" option, this behavior doesn't need to be transferred into the new componenet.

<details><summary>Code Snippet</summary>

```jsx
<Box className="form-input-select">
  <label htmlFor={'meeting-checkin'}>
    Select a meeting to check-in:
  </label>
  <Box className="radio-buttons">
    <select
      name={'meeting-checkin'}
      className="select-meeting-dropdown"
      onChange={handleEventChange}
      required
      defaultValue="--SELECT ONE--"
    >
      <option value="--SELECT ONE--" disabled hidden>
        --SELECT ONE--
      </option>
      {events.map((event) => {
        return (
          <option key={event._id || 0} value={event._id}>
            {event?.project?.name + ' - ' + event.name}
          </option>
        );
      })}
    </select>
  </Box>
</Box>
```

</details>

### Resources
- https://mui.com/material-ui/
- If you run into any questions, add a comment to the issue and tag "@trillium" 
- This issue is part of the epic: #1617

**Comment by JackHaeg** [comment]
@jng34 when you assign yourself an issue, please be sure to change the status to "in progress" (I went ahead and changed the status). Also, please only assign yourself 1 issue at a time. I recommend finishing this issue first before working on #1714, which it looks like you assigned yourself to a few days ago. 

Thanks!

**Comment by jng34** [comment]
Oops, totally forgot. Will do, moving forward. Thank you, Jack.

**Comment by JackHaeg** [comment]
Hi @jng34 I see that the PR linked to this issue is still open: https://github.com/hackforla/VRMS/pull/1840
- Usually, we allow for the GitHub automation to automatically close an issue once the linked PR is merged. In other words, we leave issues open (with the status "in progress") until the PR is merged. That way, if a PR is not merged and needs more work, the issue is still correctly displayed as "open."

I'll re-open this issue for now and will change the status back to "in progress" :)

**Comment by jng34** [comment]
@JackHaeg got it! Sorry, and thank you for being patient with me as I learn the ropes. Can I move onto another issue in the meantime?

**Comment by JackHaeg** [comment]
No worries @jng34! And yes - feel free to hop on another issue 👍
